src/appengine/handlers/cron/fuzzer_weights.py

```diff
@@ -80,48 +82,43 @@ def _coverage_formatter(query_format, dataset):
 # is indicitave of a very serious problem that makes it highly unlikely that
 # we'll find anything during fuzzing.
 STARTUP_CRASH_SPECIFICATION = QuerySpecification(
-    adjusted_weight=0.10,
-    threshold=0.80,
-    query_format=GENERIC_QUERY_FORMAT.format(field_name='startup_crash_count'),
+    query_format=GENERIC_QUERY_FORMAT.format(
+        field_name='startup_crash_count', min_weight=0.10),
     formatter=_past_day_formatter,
     reason='frequent startup crashes')
 
 # Reduce weight somewhat for fuzzers with many slow units. If a particular unit
 # runs for so long that we detect it as a slow unit, it usually means that the
 # fuzzer is not making good use of its cycles while running or needs a fix.
 SLOW_UNIT_SPECIFICATION = QuerySpecification(
-    adjusted_weight=0.50,
-    threshold=0.80,
-    query_format=GENERIC_QUERY_FORMAT.format(field_name='slow_unit_count'),
+    query_format=GENERIC_QUERY_FORMAT.format(
+        field_name='slow_unit_count', min_weight=0.50),
     formatter=_past_day_formatter,
     reason='frequent slow units')
 
 # This should end up being very similar to the slow unit specification, and is
 # included for the same reason.
 TIMEOUT_SPECIFICATION = QuerySpecification(
-    adjusted_weight=0.50,
-    threshold=0.80,
-    query_format=GENERIC_QUERY_FORMAT.format(field_name='timeout_count'),
+    query_format=GENERIC_QUERY_FORMAT.format(
+        field_name='timeout_count', min_weight=0.50),
     formatter=_past_day_formatter,
     reason='frequent timeouts')
 
 # Fuzzers which are crashing frequently may not be making full use of their
 # allotted time for fuzzing, and may end up being more effective once the known
 # issues are fixed.
 CRASH_SPECIFICATION = QuerySpecification(
-    adjusted_weight=0.50,
-    threshold=0.90,
-    query_format=GENERIC_QUERY_FORMAT.format(field_name='crash_count'),
+    query_format=GENERIC_QUERY_FORMAT.format(
+        field_name='crash_count', min_weight=0.70),
     formatter=_past_day_formatter,
     reason='frequent crashes')
 
 # Fuzzers with extremely frequent OOMs may contain leaks or other issues that
 # signal that they need some improvement. Run with a slightly reduced weight
 # until the issues are fixed.
 OOM_SPECIFICATION = QuerySpecification(
```

Comment:
nit: move OOM_SPECIFICATION block before CRASH_SPECIFICATION to show min_weight ascending order.